resources/compile-config.sh

```diff
@@ -5,19 +6,18 @@ cache_dir="$HOME/Library/Caches/com.runningwithcrayons.Alfred-2/Workflow Data/co
 mkdir -p "$cache_dir"
 
 # Get paths to configuration file
-installed_config="./config.applescript"
-cached_config="$cache_dir/config.applescript"
+installed_config="config.applescript"
```

Comment:
I realize that this will work just the same, however I still prefer the leading `./` to emphasize that this file is relative to the current working directory. Doing so also maintains consistency with the subsequent path definitions.